client/my-sites/checkout/checkout-thank-you/domains/domain-mapping-thank-you.jsx

```diff
@@ -0,0 +1,97 @@
+/**
+ * External dependencies
+ */
+
+import React from 'react';
+import { localize } from 'i18n-calypso';
+
+import { ThankYou } from 'calypso/components/thank-you';
+
+import { CALYPSO_CONTACT, SUPPORT_ROOT } from 'calypso/lib/url/support';
+import { Button } from '@automattic/components';
+import domainConnectedSuccess from 'calypso/assets/images/illustrations/domain-connected-success.svg';
+
+import './style.scss';
+
+const DomainMappingThankYou = ( { domain, translate } ) => {
+	return (
+		<ThankYou
+			containerClassName="checkout-thank-you__domains"
+			sections={ [
+				{
+					sectionKey: 'domain_mapping_whats_next',
+					sectionTitle: translate( 'What’s next?' ),
+					nextSteps: [
+						{
+							stepKey: 'domain_mapping_whats_next_plugin_setup',
+							stepTitle: translate( 'Connect your domain' ),
+							stepDescription: translate(
+								'Setup your connection by following our suggested setup.'
+							),
+							stepCta: (
+								<Button
+									href={ '#' }
+									className={ 'domain-mapping-thank-you__button' }
+									primary
+									busy={ false }
+									disabled={ false }
+								>
+									{ translate( 'Go to setup' ) }
+								</Button>
+							),
+						},
+						{
+							stepKey: 'domain_mapping_whats_next_view_posts',
+							stepTitle: translate( 'Get a profesional email' ),
```

Comment:
Please fix the spelling here. I think it may also be a bit simpler to say just "Get professional email" and drop the "a".
```suggestion
							stepTitle: translate( 'Get professional email' ),
```

Reply:
Do we really need to drop that `a`? `Get a professional email` read like a sentence, which looks fine to me.

Reply:
For me this is mostly a subtle word usage thing: it doesn't feel right to be talking about getting "a professional email" because "a professional email" is a bit ambiguous between "a professional email service/address" and "a professional email in my inbox". "Get professional email" is probably still a little bit ambiguous, but I do feel it's better. 

Maybe "Add professional email" would be better all around, especially if we go with [your suggestion to use "Add email" for the CTA](https://github.com/Automattic/wp-calypso/pull/55561#discussion_r691109045). We could also use "Set up professional email", but I think that would imply that it already existed.

I can definitely see us making this a clearer upsell around the free trials, but that can wait for a later PR.